Yafc.UI/Core/WindowMain.cs

```diff
@@ -156,6 +166,7 @@ internal override void DrawBorder(SDL.SDL_Rect position, RectangleBorder border)
         RenderingUtils.GetBorderParameters(pixelsPerUnit, border, out int top, out int side, out int bottom);
         RenderingUtils.GetBorderBatch(position, top, side, bottom, ref blitMapping);
         var bm = blitMapping;
```

Comment:
var?

Reply:
The `var`s that were replaced, were replaced only by Code Cleanup. I did not go after vars in this PR by myself.